### ISSUE TITLE
flipOutX effect doesnot work well.

### ISSUE THREAD

**Issue by amanesfera** [daneden/animate.css#370]
Most of the out effect can't work well.
Like i use the animated class "flipOutX" as the effect is completed it become disappear the html element on which i use this.

**Comment by Yeswanth-JG** [comment]
flipOutX will make the element get out of the DOM..(It will hide..!)

Please observe OUT!

**Comment by amanesfera** [comment]
@Yeswanth-JG  mate i use the chrome browser.

**Comment by daneden** [comment]
I’m not sure I follow. What exactly is the issue here?

From what I can gather, you expect the HTML element to disappear once one of the 'out' animations completes on it. This can’t be done in CSS alone.

One thing you can do is remove the element from the DOM once the animation completes, like so:

```javascript
$('#yourElement').one('webkitAnimationEnd oAnimationEnd animationend', function(){
  $(this).remove();
});
```

**Comment by amanesfera** [comment]
@daneden  Thanks i got, i use only css class but css is not enough when using any out animation class.

Thanks again for help :)

**Comment by Yeswanth-JG** [comment]
@amanesfera.. You had stated in the post 
> Like i use the animated class "flipOutX" as the effect is completed it become disappear the html element on which i use this.

Do u want to hide element after the animation or show the element after animation ends..!?

**Comment by amanesfera** [comment]
@Yeswanth-JG  it become disappear the html entity on which i use the out animation.

**Comment by Yeswanth-JG** [comment]
@amanesfera then use said `$(this).remove();` solved your issue.

**Comment by daneden** [comment]
@Yeswanth-JG @amanesfera the animation will visually hide the element from the page, but it won’t remove it from the DOM, so the element still occupies space.

`$(this).remove();` will remove the element from the page. Combined with `animationend`, this can be triggered when the animation completes.

**Comment by amanesfera** [comment]
Yes $(this).remove(); will remove the dom element

**Comment by Yeswanth-JG** [comment]
@daneden Thanx for clarifying me.!

I think we can use `$(this).detach();` incase to get the object properties to stay &attached to the element even after the element is removed!

**Comment by daneden** [comment]
@Yeswanth-JG I don’t think that’s what the issue was about—it sounds to me like [this comment](https://github.com/daneden/animate.css/issues/370#issuecomment-83050791) resolved the issue and that @amanesfera wanted to remove an element after the `flipOutX` animation completed.

Locking this issue to prevent further confusion. @amanesfera if your issue persists, open up a new one! :ok_hand:
